vespa-cloud/vespa-documentation-search/src/test/java/ai/vespa/cloud/docsearch/VespaDocSystemTest.java

```diff
@@ -111,11 +129,19 @@ private void removeTestDoc(String id) {
 
     public String getTestDoc(String id) {
         HttpResponse<String> res = testEndpoint.send(testEndpoint
-                .request("/document/v1/open/doc/docid/" + id.split(":")[4]) // id:open:doc::open/documentation/annotations.html
+                .request("/document/v1/open/doc/docid/" + id.split(":")[4])
```

Comment:
URLEncoder.encode all parts here, or at the very least the user speicified?

Reply:
(This is something many users have gotten wrong previously.)

Reply:
good point! will add